docker-compose.yml

```diff
@@ -1,10 +1,15 @@
 version: '2'
 services:
+  mysql:
+    image: mysql:latest
+    volumes:
+      - mysql:/var/lib/mysql
   limesurvey:
+    depends_on:
+      - mysql
```

Comment:
I do not believe `depends_on` will establish a `link` between containers (it's merely for controlling the lifecycle of the container in relation to other containers).

You should amend this to:

```
# ...
limesurvey:
  depends_on:
     - mysql
  links:
     - mysql
  ports: 
     - "80:80"
# ...
```

When linked, Limesurvey will need to use "mysql" as the host name:   mysql://mysql:3306